plugins/interactive/displayhandler.go

```diff
@@ -66,7 +66,7 @@ func (d *DisplayHandler) sendPings() {
 				return
 			}
 
-			d.monitor.Warn("Failed to send ping, error: ", err)
+			d.monitor.Error("Failed to send ping, error: ", err)
```

Comment:
Changed these back to error, so they'll cause tests to fail.. and then fixed all the intermittent bug!!! :)
(I hope I fixed them all!)